src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write your code here
+const numbers = document.getElementsByClassName('population');
+const totalElement = document.getElementsByClassName('total-population')[0];
+const averageElement = document.getElementsByClassName('average-population')[0];
+
+let total = 0;
+let count = 0;
+
+for (const number of numbers) {
+  total += Number(number.innerText.replace(/,/g, ''));
+  count++;
+}
+
+const average = total / count;
```

Comment:
There is a potential division by zero error here. If there are no elements with the class 'population', 'count' will be zero, leading to a division by zero when calculating the average. Consider adding a check to ensure 'count' is not zero before performing the division.